pkgs/by-name/ec/ecapture/package.nix

```
@@ -0,0 +1,129 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  stdenv,
+  lib,
+  nspr,
+  libpcap,
+  clang,
+  fd,
+  go-bindata,
+  glibc,
+  gnutls,
+  bash,
+  postgresql,
+  mariadb,
+  openssl,
+}:
+
+buildGoModule rec {
+  pname = "ecapture";
+  version = "0.8.6";
+
+  src = fetchFromGitHub {
+    owner = "gojue";
+    repo = "ecapture";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-xnUgsnz3zUkuLwqgdogEWQh0GMEmS/qmDqqmEQlHhfQ=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    clang
+    fd
+    go-bindata
+  ];
+
+  newlibpcap = libpcap.overrideAttrs (previousAttrs: {
+    configureFlags = previousAttrs.configureFlags ++ [ "--without-libnl" ];
+  });
+
+  buildInputs = [
+    newlibpcap
+    glibc.static
+    glibc
+  ];
+
+  ldflags = [
+    "-extldflags -static"
+  ];
+
+  CGO_LDFLAGS = "-lpcap -static";
+
+  doCheck = false;
+
+  hardeningDisable = [
+    "zerocallusedregs"
+  ];
+
+  patchPhase = ''
+    runHook preHook
+
+    substituteInPlace user/config/config_gnutls_linux.go \
+      --replace-fail 'return errors.New("cant found Gnutls so load path")' 'gc.Gnutls = "${lib.getLib gnutls}/lib/libgnutls.so.30"' \
+      --replace-fail '"errors"' ' '
+
+    substituteInPlace user/module/probe_bash.go \
+      --replace-fail '/bin/bash' '/run/current-system/sw/bin/bash'
+
+    substituteInPlace user/config/config_bash.go \
+      --replace-fail '/bin/bash' '/run/current-system/sw/bin/bash'
```

Just link to the bash outPath?
```suggestion
    substituteInPlace user/module/probe_bash.go \
      --replace-fail '/bin/bash' '${lib.getExe bash}'

    substituteInPlace user/config/config_bash.go \
      --replace-fail '/bin/bash' '${lib.getExe bash}'
```

---

maybe `bashInteractive` ?